frontend/www/index.html

```diff
@@ -40,9 +40,9 @@ <h1 class="title">PLAYFEST</h1>
               <input type='text' id='new-playlist-input' placeholder='Name Your Playlist' ng-model='playlistName'/>
             </label>
           </div>
-          <div id='picture' ng-controller="pictureCtrl as ctrlr">
+          <div id='picture' ng-controller="pictureCtrl as picCtrlr">
             <img ng-show="imgURI !== undefined" ng-src="{{imgURI}}">
-            <button class="button button-outline button-stable" ng-click="takePicture()">TAKE PICTURE</button>
+            <button class="button button-outline button-stable" ng-click="picCtrlr.takePicture().then(ctrl.addArrayVar)">TAKE PICTURE</button>
```

Comment:
i thinkkkk it gets confused because of the .then bit, and tries to keep looking in the same controller but im not sure i will test it out and then fix it if it works without the ctrl!! :D